web/src/app/wizard/WizardRouter.js

```diff
@@ -257,4 +188,59 @@ export default class WizardRouter extends React.PureComponent {
       return <DialogContentError error={errorMessage} />
     }
   }
+
+  return (
+    <Mutation mutation={mutation}>
+      {(commit, { data, error, loading }) => {
+        if (redirect && data && data.createService) {
+          return <Redirect push to={`/services/${data.createService.id}`} />
+        }
+
+        return (
+          <React.Fragment>
+            <Card>
+              <Form
+                onSubmit={(e, isValid) => submit(e, isValid, commit)}
+                disabled={loading}
+              >
+                <CardContent>
+                  <WizardForm
+                    disabled={status.loading}
+                    errors={fieldErrors(error)}
+                    value={value}
+                    onChange={(value) => setValue({ value })}
```

Comment:
This will break until we update the reference here as well, now that we changed the state to be flat:
```suggestion
                    onChange={(value) => setValue(value)}
```